integration_tests/mockApis/offenderSearch.ts

```diff
@@ -3,14 +3,14 @@ import { stubFor } from './wiremock'
 import { Prisoner } from '../../server/data/prisonerOffenderSearchTypes'
 
 export default {
-  getPrisoners: (
+  stubGetPrisoners: (
```

Comment:
have named all these mock APIs to start with `stub` as we had a mix and this looks to be consistent with other projects